files/zh-cn/learn/javascript/client-side_web_apis/fetching_data/index.md

```diff
@@ -186,7 +186,7 @@ Fetch API 基本上是 XHR 的一个现代替代品——它是最近在浏览
 
 #### 那么 Fetch 代码中发生了什么呢？
 
-首先，我们调用了[`fetch()`](/zh-CN/docs/Web/API/WorkerOrWindowGlobalScope/fetch)方法，将我们要获取的资源的 URL 传递给它。这相当于现代版的 XHR 中的`request.open()`,另外，您不需要任何等效的`send()`方法。
+首先，我们调用了[`fetch()`](/zh-CN/docs/Web/API/fetch)方法，将我们要获取的资源的 URL 传递给它。这相当于现代版的 XHR 中的`request.open()`,另外，您不需要任何等效的`send()`方法。
```

Comment:
```suggestion
首先，我们调用了 {{domxref("fetch", "fetch()")}} 方法，将我们要获取的资源的 URL 传递给它。这相当于现代版的 XHR 中的 `request.open()`，另外，你不需要任何等效的 `send()` 方法。
```

See line: https://github.com/mdn/content/blob/0f668c21e17896fe1f4ed4363a5f9606df36d46f/files/en-us/learn/javascript/client-side_web_apis/fetching_data/index.md?plain=1#L146

The changes we have made in translated-content should reflect the raw content in mdn/content